source/fundamentals/connection/tls.txt

```diff
@@ -211,3 +211,82 @@ object to the builder in the ``applyToSslSettings()`` lambda:
                 })
         .build();
    MongoClient client = MongoClients.create(settings);
+
+Online Certificate Status Protocol (OCSP)
+-----------------------------------------
+
+OCSP is a standard used to check whether X.509 certificates are valid or
+revoked during the TLS handshake. The driver supports the following variations
+of OCSP:
```

Comment:
suggestion: I wonder if the removal of the conjunction could simplify this?

```suggestion
OCSP is a standard used to check the validity of X.509 certificates during the TLS handshake. The driver supports the following variations
of OCSP:
```

Reply:
I think my usage of valid might not be specific enough. I was hoping that the juxtaposition of "valid" and "revoked" would signal opposites. OCSP as far as I know is only to check revocation status of a certificate with information that a revocation server can access. I don't know if revocation servers have any other responsibilities other than checking whether a certificate has been revoked prior to its expiry (whether the certificate is on the Certificate Revocation List), i.e. it doesn't explicitly validate anything about the certificate.
I found some information about the potential statuses that the revocation server can return, so I'll update.